projects/tinkerbell/rufio/patches/0001-Patch-BMC-response-interrogation.patch

```diff
@@ -0,0 +1,30 @@
+From c6aea9e215c88cab84f2235adb22fe6249b17b38 Mon Sep 17 00:00:00 2001
```

Comment:
I think we had fixed this in the latest rufio version? 
The old patch would not be needed as well.

Reply:
I think you're right. I released v0.2.0, not sure why I used v0.1.0 (which doesn't have the fix).